src/FSharpPlus/Extensions/Array.fs

```diff
@@ -6,6 +6,17 @@ module Array =
 
     open System
 
+    /// <summary>Applies an array of functions to an array of values and concatenates them</summary>
```

Comment:
Nitpicking: All sentences should end with a `.`

Reply:
Have done, and also reviewed all doc strings in Extensions.

Note: I generally treat doc summaries as a title without a full stop, however even I was inconsistent as I copied some existing comments sometimes!